src/main/webapp/join.html

```diff
@@ -0,0 +1,58 @@
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <script src="script.js"></script>
+    <script src="https://trefle.io/api/plants?q=rosemary&token=VU4zZmx5SnF3dmxVYTVsRlBTNGlqZz09"></script>
+  </head>
+  <body>
+      <nav class="navbar navbar-expand-md fixed-top nav">
+        <a class="navbar-brand" href="index.html">Plant Battles</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarCollapse">
+          <ul class="navbar-nav mr-sm-2">
+            <li class="nav-item dropdown mt-2 mt-md-0">
+                <a class="nav-link dropdown-toggle" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                Menu
+                </a>
+                <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+                <a class="dropdown-item" href="#">Profile</a>
+                <a class="dropdown-item" href="join.html">Join a Game</a>
+                <a class="dropdown-item" href="start.html">Start a Game</a>
+                <div class="dropdown-divider"></div>
+                <a class="dropdown-item" href="#">Logout</a>
+                </div>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <br>
+      <br>
+      <br>
+      <br>
+      <br>
+      <br>
+      <div class="mx-auto" style="width: 500px;">
+        <div class="card text-center" style="width: 30rem;">
+            <div class="card-body">
+              <h5 class="card-title">Join a Game</h5>
+              <form action="/join-game" method="POST">
+                  <input type="text" name="GameId" placeholder="Game Id">
+                 <input type="submit" name="submit" value="Submit">
+              </form>
+              <br>
+              <p class="card-text">Want to join an already existing game? Submit the game ID here!</p>
+              <br>
+            </div>
+        </div>
+      </div>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
```

Comment:
do u have to put the bootstrap scripts at the bottom of the page? I put mine in the header and I think it still works.

Reply:
I put them in the bottom following the tutorial they had, but I think you're right they do work in the top and they look better there

Reply:
https://stackoverflow.com/questions/30488298/where-in-the-html-file-should-jquery-and-bootstrap-be-placed

For reasoning on having something at the top vs the bottom.